doc/compiler/il/SymbolsSymrefsAliasing.md

```diff
@@ -164,51 +164,26 @@ associated with queries that compose the larger query.
 ### TR_AliasSetInterface
 
 The `TR_AliasSetInterface` class is a wrapper around a bit vector representing aliased
-symrefs. It provides various methods for performing common operations on these bit vectors.
-[The CRTP](https://en.wikipedia.org/wiki/Curiously_recurring_template_pattern) is used
-to achieve static polymorphism.
+symrefs. It provides various methods for performing common operations on these bit vectors. 
+The non-type template parameter is one of the two values of
+the `AliasSetInterface` enum: `UseDefAliasSet` or `UseOnlyAliasSet`.
 
-One key interface method that *must* be overridden by sub-classes is `getTRAliases()`.
-This is the method responsible for returning the bit vector of aliases.
+One key interface method is `getTRAliases()`. This template specialized method is responsible for 
+returning the bit vector of aliases.
+
+The class can also be initialized from a `TR::Node` instance through `Node::mayKill()` or 
+`Node::mayUse()` method. It provides an API for dealing with aliases of the symref *contained* by 
+the node. If the`TR::Node` instance calls `mayKill()` or `mayUse()` method without a `symref`, it 
+will initialize an empty `AliasSetInterface` and `getTRAliases()` will return an empty bit container.
```

Comment:
Might be good to clarify this point a bit more, e.g. does it return the "same" empty bit container every time or it allocates a new one ?

Reply:
Sorry about my unclear description there. 
If symref is empty (NULL pointer), the `getTRAliases()` is going to  return a NULL pointer. I have changed my wording in the new commit.